flight/Modules/Autotune/autotune.c

```diff
@@ -50,28 +50,18 @@
 #include "pios_thread.h"
 #include "systemsettings.h"
 
-#include "circqueue.h"
 #include "misc_math.h"
 
 // Private constants
-#define STACK_SIZE_BYTES 1340
+#define STACK_SIZE_BYTES 640
```

Comment:
The plan is for the task to go away, and it's not much more work to do so.  I just wimped out and decided to go this way for now.